google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineAction.java

```diff
@@ -104,7 +104,7 @@ CloudSdk prepareExecution(
         .startListener(new ProcessStartListener() {
```

Comment:
I think this would look cleaner if you defined the ProcessStartListener as a variable earlier so as not to break the flow of the builder.